docker/puppetserver-standalone/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:16.04
+
+ARG vcs_ref
+ARG build_date
+ENV PUPPET_SERVER_VERSION="5.3.1"
+ENV DUMB_INIT_VERSION="1.2.1"
+ENV UBUNTU_CODENAME="xenial"
+ENV PUPPETSERVER_JAVA_ARGS="-Xms256m -Xmx256m"
+ENV PATH=/opt/puppetlabs/server/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:$PATH
+ENV PUPPET_HEALTHCHECK_ENVIRONMENT="production"
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppetserver" \
+      org.label-schema.name="Puppet Server (No PuppetDB)" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version=$PUPPET_SERVER_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppetserver" \
+      org.label-schema.vcs-ref=$vcs_ref \
+      org.label-schema.build-date=$build_date \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
+
+RUN apt-get update && \
```

Comment:
This is the common, accepted format for RUN scripts inside Dockerfiles, so my approach is non-standard. With that said, I still think copying in a script from source code and running that has lower maintenance costs and a reduced chance of intermittent bugs. 

COPY srcbin/install_puppet /install_puppet
RUN /install_puppet

Reply:
I don't want to do any major refactoring as part of this initial work, I'm just trying to get the existing automation to a state where it works in the project repos and can have pipelines setup for auto publishing.